nss-day-cohort-39/tabloid-full-stack-del_rios
# Db jr categories

### Changes
- added CategoryProvider, Category.js, CategoryList.js, CategoryController.cs, CategoryRespository.cs, Category.cs

### Testing
- pull down branch
- confirm link in navbar displays category list
- confirm "add category" button displays input field and save category adds it to the list
- confirm edit button allows you to edit a category name and save it
- confirm that cancel button on edit modal does not save any changes made
- confirm delete button displays modal
- confirm yes actually deletes category 
- confirm cancel closes modal and does not delete category
